cdk/kittyhawk/src/application/base.ts

```diff
@@ -36,6 +36,7 @@ export class Application extends Construct {
     // We want to prepend the project name to the name of each component
     const release_name = process.env.RELEASE_NAME || "undefined_release";
     const fullname = `${release_name}-${appname}`;
+    const isFeatureBranch = process.env.IS_FEATURE_BRANCH == "true";
```

Comment:
Instead of checking for the environment variable for `isFeatureBranch` everywhere, would be better to have this as an importable variable in a utils file. 

Also, the name `isFeatureBranch` is not very good because it is unclear what "is" is referring to. A better name would be having the environment variable `DEPLOY_TO_FEATURE_BRANCH`.

The naming convention is also inconsistent: if release_name is in snake case, then so should this.

Reply:
Moving it to a variable seems to make it be always false.